test/compress/issue-12.js

```diff
@@ -10,8 +10,8 @@ keep_name_of_setter: {
     options = {
         unused: true,
     }
-    input: { a = { set foo () {} } }
-    expect: { a = { set foo () {} } }
+    input: { a = { set foo (v) {} } }
+    expect: { a = { set foo (v) {} } }
```

Comment:
While you're here, could you make these standard beautify'd blocks?

Reply:
Done in https://github.com/mishoo/UglifyJS/pull/4652/commits/db7d53623891aa0d6bc3a9ed9243f14d5a757822